tests/python/unit/test_symspell_checker.py

```diff
@@ -6,18 +6,20 @@
 import os
 import pandas as pd
 
+from spacy_tokenizer import MultilingualTokenizer  # noqa
 from symspell_checker import SpellChecker  # noqa
 
 dictionary_folder_path = os.getenv("DICTIONARY_FOLDER_PATH")
+stopwords_folder_path = os.getenv("STOPWORDS_FOLDER_PATH")
```

Comment:
Secure this call and check on environment variables, are they defined, do they refer to existing file ?

Reply:
Same comment, I want to check for this, it is defined in the makefile